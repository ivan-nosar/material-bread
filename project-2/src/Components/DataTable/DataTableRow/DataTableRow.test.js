import React from 'react';
import { DataTableRow } from '../../..';

import renderer from 'react-test-renderer';

test('DataTableRow Renders', () => {
  const tree = renderer.create(<DataTableRow />).toJSON();
  expect(tree).toMatchSnapshot();
});
