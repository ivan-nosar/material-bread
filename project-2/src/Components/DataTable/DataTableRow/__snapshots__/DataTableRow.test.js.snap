// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataTableRow Renders 1`] = `
<View
  accessible={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Array [
      Object {
        "alignItems": "stretch",
        "borderBottomWidth": 0.5,
        "borderStyle": "solid",
        "flexDirection": "row",
        "minHeight": 48,
        "minWidth": "auto",
      },
      Object {
        "backgroundColor": "transparent",
        "borderBottomColor": "rgb(224, 224, 224)",
      },
      undefined,
    ]
  }
/>
`;
